kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -160,10 +154,9 @@ class SolvingStrategy
      * @param rModelPart The model part of the problem
      * @param ThisParameters The configuration parameters
      */
-    virtual typename ClassType::Pointer Create(
+    typename ClassType::Pointer Create(
```

Comment:
I think this needs to be virtual

Reply:
I did it virtual in the very first stage but it didn't work. I think that the problem is that the implicit strategy has different number of template arguments, then we cannot point with a `SolvingStrategy::Pointer` to an `ImplicitSolvingStrategy`.

Reply:
So you may have an empty linear solver passed to the explicit solver by default

Reply:
Done. Finally the solution was as simple as defining an extra type for the base class :facepalm: